iOS/rainbow/Controller/MainTabBarController.swift

```diff
@@ -60,11 +60,11 @@ class MainTabBarController: UITabBarController {
             
             ScoreEntry.ServerCalls.save(entry: newEntry, completion: { entry, error in
                 if error != nil {
-                    SVProgressHUD.showError(withStatus: "Could not save user")
+                    SVProgressHUD.showError(withStatus: "You'll need an internet connection to get on the leaderboard!")
```

Comment:
could this error message be like: "Error getting response from server. Please try again"

Reply:
The key element I want to express is that the game can still be played. @anamita what do you think?

Reply:
That would be when someone taps `view leaderboard`

Reply:
this happens when you start the game and end the game. May be we could say " there is error but keep playing"

Reply:
What about "Couldn't connect to the server - keep playing!"